src/layout/FileUploadWithTag/FileUploadWithTagComponent.test.tsx

```diff
@@ -160,12 +160,6 @@ describe('FileUploadWithTagComponent', () => {
         component: { maxNumberOfAttachments: 3 },
         attachments: getAttachments({ count: 2 }),
       });
-
```

Comment:
I think we still need these tests. 😄

Reply:
Yeah I was wondering about that, but aren't they testing the same as the script below them (getByTestId and queryByTestId)?